packages/expo-modules-core/android/src/main/java/expo/modules/core/interfaces/ReactNativeHostHandler.kt

```diff
@@ -48,4 +50,16 @@ interface ReactNativeHostHandler {
     useDeveloperSupport: Boolean
   ) {
   }
+
+  /**
+   * Callback before {@link ReactInstanceManager} creation
+   */
+  fun onBeforeCreateReactInstanceManager(useDeveloperSupport: Boolean) {}
```

Comment:
The usage of `before` sounds kinda odd, especially there is `did` (not `after`) in the other callback. Recently I've found these guidelines from Microsoft for .NET: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-events
and I started thinking about our own naming conventions. Should we follow Microsoft, or maybe Apple with `will` and `did` to differentiate actions before and after? What do you guys think? cc @brentvatne

Reply:
good catch! it was actually before/after, but later i changed it to "did" but not update the "before". i think will/did make sense to me. will change to will/did first.